src/blog/2023/08/flowforge-1-10-release.md

```diff
@@ -0,0 +1,82 @@
+---
+title: FlowForge now available on  AWS Marketplace
+subtitle: New FlowForge 1.10 also includes improvements to device management and importing environment variable templates.
+description: New FlowForge 1.10 release makes is possible to install the FlowForge platform from the AWS marketplace and improvements to device management.
+date: 2023-08-03 
+authors: ["ian-skerrett"]
+image: /blog/2023/07/images/release-1.9.0.png
+tags:
+    - posts
+    - flowforge
+    - releases
+---
+
+FlowForge 1.10 now makes it possible to install the FlowForge platform from the AWS Marketplace. In addition, the release includes improvements to device management and importing environment variable templates.
+
+<!--more-->
+
+## AWS Marketplace
+It is now possible to install FlowForge on AWS directly from the AWS Marketplace. This makes it a lot easier for developers to install and run FlowForge on the AWS Cloud. 
+
+The FlowForge open source edition is the default version offered from the marketplace, so users can get started free of charge.  Users are able to use the BYOL (Bring Your Own License) feature of AWS Marketplace to upgrade to a commercially licensed version.
+
+
+## Import Environment Variable Templates  [#2372](https://github.com/flowforge/flowforge/issues/2372)
+
+FlowForge 1.10 now allows users to import environment variable templates. This makes it much easier and less error prone to maintain and add new environment variables to Node-RED instances running on FlowForge. 
+
+
+## DevOps Pipelines now can include devices [#2243](https://github.com/flowforge/flowforge/issues/2243)
+DevOps pipelines have proven very popular for creating dev/test/production environments for Node-Red flow development. Now, devices can be associated with a pipeline so when a snapshot is created it can be pushed to all the devices associated with the pipeline.  This will improve the overall quality and reliabiliy of Node-RED development for remote devices.
```

Comment:
```suggestion
DevOps pipelines have proven very popular for creating dev/test/production environments for Node-Red flow development. Now, devices can be associated with a pipeline, so that when an instance is deployed, changes can also be deployed to all devices connected to that stage's instance, and a snapshot will be created automatically. This will improve the overall quality and reliability of Node-RED development for remote devices.
```